### ISSUE TITLE
Crashing on opening and closing NWB format files in File Reader

### ISSUE THREAD

**Issue by anjaldoshi** [open-ephys-plugins/nwb-format#1]
In Release mode: Opening a NWB format recording in the File Reader is working fine, but crashes when File Reader is deleted or when GUI is closed.

In Debug mode: Opening a NWB Format file in File Reader throws exception and crashes.

Debug Call Stack:

![image](https://github.com/open-ephys-plugins/nwb-format/assets/11559287/ac5263dc-80dc-4dee-87e7-218ed0f2669e)

**Comment by anjaldoshi** [comment]
Re-installing the latest version of the plugin fixed the issue  for Release mode. Debug mode exception can be figured out once HDF5 debug libraries are built.  Closing this for now.

**Comment by bgrgic-inbrain** [comment]
Hi @anjaldoshi,

have you tried using nwb-format plugin with Open Ephys v1.0+? I am getting the same errors described here, trying to deallocate corrupt memory like shown in the stack trace above.

I loaded the standard File Reader recording, added a record node, recorded in .nwb format then removed the record node and tried to load the same file with the File Reader. This is when I built the source for OpenEphysHDF5Lib and NWB Format.

When I install the NWB Format from the plugin installer, it does work with the example dataset but is distorting data with my custom recordings.
